tests/patch-series.test.ts

```diff
@@ -423,7 +423,8 @@ Fetch-It-Via: git fetch ${repoUrl} my-series-v1
                 "some description goes here",
                 "",
                 "Cc: Some Contributor <contributor@example.com>",
-                "CC: Capital Letters <shout@out.loud>",
+                "CC: Capital Letters <shout@out.loud>, Hello <hello@wor.ld>"
+                + ", without@any.explicit.name",
                 "Cc: Git Maintainer <maintainer@gmail.com>",
```

Comment:
Add a test with a semicolon delimiter?  It is supported by addressparser() so maybe we want to know if support is removed.  Some people may be used to using semicolon.

Reply:
Good idea! I added the test (the code did not need any change, as you probably expected: `addressparser()` does all the hard work for us here.

For future reference: if we ever introduce that mapping from GitHub handles to email addresses, we would want to introduce that handling also in the code in the `patch-series.ts` file that was touched above, allowing e.g. `Cc: @dscho` and/or `Cc: dscho`.

Reply:
Spaces vs tabs in line 428?

Reply:
Oops, sorry. Force-pushed a fix.